content/en/docs/concepts/storage/volumes.md

```diff
@@ -255,11 +255,40 @@ If that is filled up from another source (for example, log files or image
 overlays), the `emptyDir` may run out of capacity before this limit.
 
 {{< note >}}
-If the `SizeMemoryBackedVolumes` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled,
-you can specify a size for memory backed volumes.  If no size is specified, memory
+The `SizeMemoryBackedVolumes` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled by default since v1.22,
+so you can specify a size for memory backed volumes.  If no size is specified, memory
 backed volumes are sized to node allocatable memory.
```

Comment:
Try this:
```suggestion
You can specify a size for memory backed volumes, provided that the
`SizeMemoryBackedVolumes` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
enabled in your cluster (this has been beta, and active by default, since the Kubernetes 1.22 release).
If you don't specify a volume size, memory
```